Cli/Command/AnalysisCommand.php

```diff
@@ -28,6 +28,7 @@ protected function configure()
             ->addArgument('project-uuid', InputArgument::REQUIRED)
             ->addOption('format', null, InputOption::VALUE_REQUIRED, 'To output in other formats', 'txt')
             ->addOption('fail-condition', null, InputOption::VALUE_REQUIRED, '')
+            ->addOption('show-ignored-violations', 's', InputOption::VALUE_NONE, 'Show ignored violations')
```

Comment:
I would not add `-s` because I think it won't be used that much but also because `-s` means silent, which is the contrary of what we want here :)